app/models/legal_aid_application.rb

```diff
@@ -401,8 +401,12 @@ def default_cost_limitation
     default_substantive_cost_limitation
   end
 
+  def display_emergency_certificate?
+    used_delegated_functions? && !special_children_act_proceedings?
+  end
+
   def substantive_cost_overridable?
-    substantive_cost_limitation.present? && default_substantive_cost_limitation < MAX_SUBSTANTIVE_COST_LIMIT
+    substantive_cost_limitation.present? && default_substantive_cost_limitation < MAX_SUBSTANTIVE_COST_LIMIT && !special_children_act_proceedings?
```

Comment:
Does this need to be added? We only seem to be calling the new `display_emergency_certificate?` below?

Reply:
Not sure that I understand? (probably missing something!) We are still calling substantive_cost_overridable in multiple places emergency_cost_override_form, limitations_controller, checked_provider_ansers/shared.html.erb, limitations/cost_limit.html.erb, merits_reports/show.html.erb and review_application/questions_and_answers.html.erb. The display_emergency_certificate was intended to be the logic to determine whether or not to display the emergency certificate which Sara said we shouldn't do for SCA applications. The substantive_cost_overridable was intended to be the logic whether or not the provider can override the substantive cost limit.

Reply:
🤦‍♂️ In that case, feel free to ignore, this is fine!